src/main/java/com/alkemy/ong/entity/CategoryEntity.java

```diff
@@ -11,7 +11,6 @@
 @Entity
 @Getter @Setter @ToString @RequiredArgsConstructor
 @SQLDelete(sql="UPDATE categorys SET softDelete = true WHERE id=?")
```

Comment:
I suggest the table's name should be categories, that is the plural(here is says categorys). If we refactor the table's name, we need to take into account to drop a create the DB afterwards

Reply:
We will agree on it, and we will modify it if it is so.I agree with your way of seeing it.